### PR TITLE
added snip for random number genetor

### PR DESCRIPTION
**Pull request by savi001**
added snip for randon number generator within a given range, please specify if any change is required

### CODE REVIEW

**Comment by DevTMK** [comment]
Hey @savi001, you are supposed to create a new snip in the src folder.
Take a look at [contributing guidelines](https://github.com/DevTMK/snips/blob/9661c2f0362c46d8383ef5a5a18f0a557052dfde/CONTRIBUTING.md).

File: RandomNumberGenerator.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+```
+  // Add final result
+```
+
+## Snip Langauge
+* [JavaScript](#javascript)
+
+### JavaScript
+```js
+  <script>
```
Comment (DevTMK) [inline RandomNumberGenerator.md:14]: Just keep the function, remove other code

**Review by DevTMK — CHANGES_REQUESTED** [review]

**Comment by savi001** [comment]
what should i have to write in "Add final result here" section

File: RandomNumberGenerator.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# [Random Number Generator within a given Range]
+[Its an snippet for a  function random number generator which whill generates a number within a given range include extreme values]
+
+## Result
+```
+console.log(randomNumber( 1, 100 )); // 68
```
Comment (DevTMK) [inline RandomNumberGenerator.md:6]: It should be -
randomNumber(1, 100); // 68

**Comment by savi001** [comment]
see if any futher change is required

**Review by DevTMK — APPROVED** [review]
Thanks @savi001, Merging

**Pull request merged**